packages/sync-actions/src/order-actions.js

```diff
@@ -37,3 +38,56 @@ export function actionsMapDeliveries(diff, oldObj, newObj) {
 
   return handler(deliveriesDiff, oldObj.shippingInfo, newObj.shippingInfo)
 }
+
+export function actionsMapReturnsInfo(diff, oldObj, newObj) {
+  const returnInfoDiff = diff.returnInfo
+  if (!returnInfoDiff) return []
+
+  const handler = createBuildArrayActions('returnInfo', {
+    [CHANGE_ACTIONS]: (oldSReturnInfo, newReturnInfo) => {
+      const updateActions = Object.keys(returnInfoDiff).reduce(
+        (itemActions, key) => {
+          const { items = {} } = diff.returnInfo[key]
```

Comment:
nitpick: I think we could also use `returnInfoDiff` variable here, so
`const { items = {} } = returnInfoDiff[key]`